beluga/include/beluga/sensor/data/occupancy_grid.hpp

```diff
@@ -34,6 +34,9 @@
 
 namespace beluga {
 
+/// General value traits
+enum class GridStatus : std::uint8_t { kFree, kUnknown, kOccupied };
```

Comment:
@DPR00 do we need to pollute the `beluga` namespace with this, or can we nest it within the class?